src/main/kotlin/io/ipfs/kotlin/commands/Add.kt

```diff
@@ -40,9 +60,17 @@ class Add(val ipfs: IPFSConnection) {
                 .build();
 
         val response = ipfs.okHttpClient.newCall(request).execute().body()
-        val result = adapter.fromJson(response.source())
+        var result: NamedHash = NamedHash("nameprobe","hashprobe")
+
+//        val result = adapter.fromJson(response.source())
```

Comment:
I tried to unwind the WIP and debugging but unfortunately they were intermixed, so those lines aren't in the final commits. It's much easier to read in "Files changed" in any case.